yodeploy/cmds/build_artifact.py

```diff
@@ -8,16 +8,21 @@
 import socket
 import subprocess
 import sys
-import urllib2
 from xml.etree import ElementTree
 
+try:
+    from urllib.request import Request, urlopen
+    from urllib.error import URLError
+except ImportError:
+    from urllib2 import Request, urlopen, URLError
+
 sys.path.insert(0, os.path.join(os.path.dirname(__file__), '..', '..'))
 
-from yoconfigurator.base import write_config
-from yoconfigurator.filter import filter_config
-from yoconfigurator.smush import config_sources, smush_config
-import yodeploy.config
-import yodeploy.repository
+from yoconfigurator.base import write_config  # noqa
+from yoconfigurator.filter import filter_config  # noqa
+from yoconfigurator.smush import config_sources, smush_config  # noqa
+import yodeploy.config  # noqa
+import yodeploy.repository  # noqa
```

Comment:
Yeah, it's this bit from pep8:

> Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants.

That doesn't allow for path manipulation like we're doing here and, in general, I think it's a good rule. I imagine snitch would be similarly annoyed without the noqaing and I guess if I weren't deleting the lint comments... ⚡️